newIDE/app/src/MainFrame/index.js

```diff
@@ -789,63 +789,6 @@ const MainFrame = (props: Props) => {
     toolbar.current.setEditorToolbar(editorToolbar);
   };
 
-  const addLayout = () => {
```

Comment:
nice! weird we didn't catch this duplication before 🤔

Reply:
Yeah 😅 Was long for a long time I guess.